_load_data.R

```diff
@@ -31,12 +31,15 @@ bblcentroids <- read_csv(
 # Read BIN centroids data, make them distinct on the BIN key
 bincentroids <- read_csv(
   paste0(dataDir, "/bincentroids.csv"),
-  col_types = cols(
-    bin = col_character()
+  col_types = cols_only(
+    BIN = col_character(),
```

Comment:
is this now `BIN` uppercase that's how it is in the new dataset?

Reply:
yes, that's how the CSV comes from opendata. I update the column name to `bin` to ensure downstream compatibility.